### PR TITLE
installation docs: add minimum container-selinux version

### PR DESCRIPTION
**Pull request by jean-edouard**

### CODE REVIEW

**Review by xpivarc — COMMENTED** [review]
I think this will be hard to version so I would prefer to have this information in Kubevirt main repo. WDYT?

**Comment by jean-edouard** [comment]
> I think this will be hard to version so I would prefer to have this information in Kubevirt main repo. WDYT?

That's a good point, do you have a specific location in mind?
What do you think about keeping this PR too with a link to that kubevirt/kubevirt file so people are made aware of it when they follow the docs?

**Comment by jean-edouard** [comment]
Reworded this and added https://github.com/kubevirt/kubevirt/pull/8580.
Will close this PR if deemed redundant.
@xpivarc PTAL!

File: docs/operations/installation.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ virtualization workloads:
       QEMU: Checking if device /dev/net/tun exists                               : PASS
     ...
 
+### SELinux support
+
+SELinux-enabled nodes need [Container-selinux](https://github.com/containers/container-selinux) installed. The minimum version is documented inside the kubevirt/kubevirt repository, in docs/getting-started.md, under "SELinux support".
+
+For (older) release branches that don't specify a container-selinux version, version 2.170.0 or newer is required.
```
Comment (xpivarc) [inline docs/operations/installation.md:48]: `required` -> `recommended`? I mean at least for me it is hard to say what we require in X(pick a random one :) ) without a deeper look.

**Review by xpivarc — COMMENTED** [review]
/lgtm

**Comment by cwilkers** [comment]
I think it's valuable to have the information in the user-guide pointing at the kubevirt/kubevirt docs. This will help searchability.

/approve

**Comment by rmohr** [comment]
> I think it's valuable to have the information in the user-guide pointing at the kubevirt/kubevirt docs. This will help searchability.
> 
> /approve

+1 on that. If it is in the kubevirt main repo, then it is separate from the install guide which users will use. Also the release cadence is now much slower, decreasing the chance that this gets out of sync, since we have quite some time to update the user-guide in a new release.

**Pull request merged**